Application/EdFi.Ods.AdminApi/Features/RequestLoggingMiddleware.cs

```diff
@@ -76,9 +77,15 @@ public async Task Invoke(HttpContext context, ILogger<RequestLoggingMiddleware>
                     await response.WriteAsync(JsonSerializer.Serialize(notFoundResponse));
                     break;
 
+                case IAdminApiException adminApiException:
+                    logger.LogError(JsonSerializer.Serialize(new { message = "An uncaught error has occurred", error = new { ex.Message, ex.StackTrace }, traceId = context.TraceIdentifier }));
+                    response.StatusCode = adminApiException.StatusCode.HasValue ? (int)adminApiException.StatusCode : 500;
+                    await response.WriteAsync(JsonSerializer.Serialize(new { message = "The server encountered an unexpected condition that prevented it from fulfilling the request." }));
+                    break;
+
                 default:
                     logger.LogError(JsonSerializer.Serialize(new { message = "An uncaught error has occurred", error = new { ex.Message, ex.StackTrace }, traceId = context.TraceIdentifier }));
-                    await response.WriteAsync(JsonSerializer.Serialize(new { message = ex?.Message }));
+                    await response.WriteAsync(JsonSerializer.Serialize(new { message = "The server encountered an unexpected condition that prevented it from fulfilling the request." }));
```

Comment:
Previously, we were sending a detailed error message to the API client, which should never occur.